mappings/net/minecraft/client/render/BufferBuilderStorage.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4599 net/minecraft/client/render/BufferBuilderStorage
+	FIELD field_20956 blockBuilders Lnet/minecraft/class_750;
+	FIELD field_20957 otherBuilders Ljava/util/SortedMap;
+	FIELD field_20958 generalDrawer Lnet/minecraft/class_4597$class_4598;
+	FIELD field_20959 blockBreakingProgressDrawer Lnet/minecraft/class_4597$class_4598;
+	FIELD field_20961 teamColorAwareOutlineDrawer Lnet/minecraft/class_4618;
+	METHOD method_22997 getBlockBuilders ()Lnet/minecraft/class_750;
+	METHOD method_23000 getGeneralVertexConsumers ()Lnet/minecraft/class_4597$class_4598;
+	METHOD method_23001 getBlockBreakingVertexConsumers ()Lnet/minecraft/class_4597$class_4598;
+	METHOD method_23003 getTeamColorAwareOutlineVertexConsumers ()Lnet/minecraft/class_4618;
```

Comment:
Consider improving the name of this method. I recall you renamed this somewhere else